builder/js/ncf.js

```diff
@@ -89,6 +89,9 @@ app.directive('constraint', function($http, $q, $timeout) {
     },
     require: 'ngModel',
     link: function(scope, elm, attrs, ctrl) {
+    if((scope.parameter.constraints.allow_empty_string)&&(scope.parameter.value==null)){
```

Comment:
I thought it was better to check both cases, null and undefined. But ok, it's done!